_pages/01-applicants.md

```diff
@@ -112,4 +112,14 @@ please mention that in the application.
 
 Yes. While we understand that not all topics may be 100% relevant to you we want to maximize engagement. Note that we post the lessons and videos online later, so if you are interested in only a portion of the OceanHackWeek consider that an option rather than taking someone's spot.
 
----
+**Q: I did not get accept, what now?**
+
+Sadly resources are limited and we cannot accommodate all the applicants we would like.
+
+It is important to note that a rejected application does not mean a bad one! The reasons are:
+
+- The organizer tries to accommodate a good balance between novices, intermediaries and experts. That means a candidate may be rejected one year and accepted in another due to the different pool of applicants. Rule of thumb? Always apply again if possible! Also, if you believe you can contribute to the OHW as a helper instead of a student, please get in touch! We are always in need of helpers.
+
+- One important factor that goes into the selection of candidates is the motivation paragraph in your application. Boilerplate answers or short answers like: "I want to participate" or "It is important to me" may get lower ranking than those who read the past year materials and wrote a paragraph on how the OHW can really make a difference to them. TL;DR be honest, be you!
```

Comment:
Here's a potential suggested changed that I'm not 100% sold on: "In short: be informed, be honest, be you!". It sounds more intimidating than what you have there, but it's also more helpful and does reflect what's said in the longer text. What do you think?

I've also changed the "TL;DR" bit. I don't think that lingo is universally known.

Reply:
I don't think that it is intimidating. Let's add that.